dataflow-website/recipes/scaling/kubernetes/prometheus/prometheus-configmap.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus
+  labels:
+    app: prometheus
+data:
+  alert.rules.yml: |-
+    groups:
+      - name: scdfrules
+        rules:
+          - alert: HighThroughputDifference
+            expr: avg(irate(spring_integration_send_seconds_count{name!="errorChannel",name!="nullChannel",result="success",type="channel",application_name="time"}[1m])) by(stream_name)  - avg(irate(spring_integration_send_seconds_count{name!="errorChannel",name!="nullChannel",result="success",type="channel",application_name="transform"}[1m])) by(stream_name) > 100
+            for: 30s
+            annotations:
+              summary: "Throughput delta displace for {{ $labels.stream_name }}"
```

Comment:
Nitpick: I don't understand the use of "displace" - perhaps change it to something easy to grok

Reply:
Ah, thought i've fixed those. it should be `Throughput delta difference`